### PR TITLE
LanguageToolBear: Use en-US on invalid lang guess

### PR DESCRIPTION
**Pull request by pareksha**
This makes LanguageToolBear throw an warning and use
'en-US' if invalid language is guessed by ``language_check``.

Fixes https://github.com/coala/coala-bears/issues/1848

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by pareksha** [comment]
@Makman2, changes made in reference to https://github.com/coala/coala-bears/issues/1848#issuecomment-361059148. Please review.

File: bears/natural_language/LanguageToolBear.py [hunk 1]
```diff
@@ -57,12 +59,18 @@ def run(self,
...
-                           else natural_language
 
-        tool = LanguageTool(natural_language, motherTongue='en_US')
-        tool.disabled.update(languagetool_disable_rules)
+        try:
+            tool = LanguageTool(natural_language, motherTongue='en_US')
+        except ValueError:
+            logging.warn(
+                "Changing the `natural_language` setting to 'en-US' as "
+                "`language_check` failed to guess a valid language'."
```
Comment (Makman2) [inline bears/natural_language/LanguageToolBear.py]: I believe `'` was intended to be removed ;)

Comment (Makman2) [reply bears/natural_language/LanguageToolBear.py]: (then you can use single quotes again for this line of string)

File: bears/natural_language/LanguageToolBear.py [hunk 2]
```diff
@@ -45,7 +46,8 @@ def run(self,
         :param natural_language:           A locale representing the language
                                            you want to have checked. If set to
                                            'auto' the language is guessed.
-                                           If the language cannot be guessed,
+                                           If the language cannot be guessed
+                                           or an invalid language is guessed,
```
Comment (Makman2) [inline bears/natural_language/LanguageToolBear.py]: not "or an invalid language is provided"? I mean an invalid language can't be guessed, either the guess works or it fails with a `ValueError`, or not?

Comment (pareksha) [reply bears/natural_language/LanguageToolBear.py]: Ya, so I am changing ``invalid language`` to ``unsupported language`` as the language is not supported by ``language_check``.

File: bears/natural_language/LanguageToolBear.py (hunk 1 above)
Comment (pareksha) [reply bears/natural_language/LanguageToolBear.py]: Sorry bad mistake :worried:

**Comment by pareksha** [comment]
@Makman2 changes made. Review requested. :smiley:

File: bears/natural_language/LanguageToolBear.py [hunk 3]
```diff
@@ -57,12 +59,19 @@ def run(self,
...
                             if natural_language == 'auto'
                             else natural_language)
-        natural_language = 'en-US' if not natural_language \
-                           else natural_language
 
-        tool = LanguageTool(natural_language, motherTongue='en_US')
-        tool.disabled.update(languagetool_disable_rules)
+        try:
+            tool = LanguageTool(natural_language, motherTongue='en_US')
+        # Using 'en-US' if guessed language is not supported
```
Comment (Makman2) [inline bears/natural_language/LanguageToolBear.py]: I would put the comment into the `except`-block, so just move it one line down :) That's a more common placement :)

**Comment by pareksha** [comment]
@Makman2 changes made. Please review :smile:

**Comment by Makman2** [comment]
ack dd1d75d65d04c60f9caa2dca85453e8231a33882

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**